.gitignore

```
@@ -53,4 +53,4 @@ build/
 build-debug/
 build-release/
 cmake-build-debug/
-cmake-build-release/
+cmake-build-release/
```

I would skip making any changes to the file.


```suggestion
cmake-build-release/

```

---

Done

---

My bad. I should have noticed this diff and removed it.